Gemfile

```diff
@@ -173,3 +173,5 @@ gem 'net-ssh'
 gem 'net-scp'
 gem 'httparty'
 gem 'jquery-cookie-rails'
+gem 'oj'
```

Comment:
Is the optimized JSON gem required for this PR?   

I'm a little nervous about taking an external, third-party dependency on something as fundamental as JSON parsing if it's not really important.